src/functionalTest/java/uk/gov/hmcts/reform/pip/data/management/controllers/lists/PublicationCareStandardsListTest.java

```diff
@@ -45,6 +46,7 @@
 @WithMockUser(username = "admin", authorities = { "APPROLE_api.request.admin" })
 class PublicationCareStandardsListTest {
 
+    @Qualifier("artefact")
```

Comment:
Sorry I should have thought of this earlier when talking about beans. 

In spring you can use the @Primary annotation on a bean to make it the default choice. We could do this in the AzureBlob config which would then remove the need to add @Qualifier to all of these

Reply:
Done